lms/static/scripts/frontend_apps/components/dashboard/test/DashboardActivityFilters-test.js

```diff
@@ -64,7 +64,8 @@ describe('DashboardActivityFilters', () => {
   let fakeLoadNextCoursesPage;
   let fakeLoadNextAssignmentsPage;
   let fakeLoadNextStudentsPage;
-  let wrappers = [];
+  let wrappers;
+  let containers;
```

Comment:
Yep, that would be nice.

In fact, we should probably check if it's possible to always attach tested components to the DOM, without breaking too many tests and making tests much slower.

Reply:
> In fact, we should probably check if it's possible to always attach tested components to the DOM, without breaking too many tests and making tests much slower.

I actually prefer to run as many tests as possible on disconnected elements, since there is no way that disconnected elements can affect each other or subsequent tests. Disconnected nodes should be much cheaper, since they can avoid a lot of work, but it would be interesting to actually measure.

Reply:
> since there is no way that disconnected elements can affect each other or subsequent tests

That's a good point.

Reply:
> In future we could reduce boilerplate in individual apps by adding a utility in @hypothesis/frontend-shared to assist with this.

BTW, I guess you mean `@hypothesis/frontend-testing`?

Reply:
Yes, indeed!